plugins/modules/ipa_dnsrecord.py

```diff
@@ -51,6 +52,7 @@
     - In the case of 'A6' record type, this will be the A6 Record data.
     - In the case of 'CNAME' record type, this will be the hostname.
     - In the case of 'DNAME' record type, this will be the DNAME target.
+    - In the case of 'NS' record type, this wil be the name server hostname. Hostname must already have a valid A or AAAA record.
```

Comment:
```suggestion
    - In the case of 'NS' record type, this will be the name server hostname. Hostname must already have a valid A or AAAA record.
```